rts/lra/lra-coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/LRARecord.java

```diff
@@ -555,6 +566,50 @@ private int retryGetEndStatus(URI endPath, boolean compensate) {
         return -1;
     }
 
+    private Future<Response> getAsyncResponse(WebTarget target, String method, AsyncInvoker asyncInvoker, String compensatorData, String mediaType) {
+        String queryString = target.getUri().getQuery();
+
+        if (queryString != null) {
+            String[] queries = queryString.split("&");
+
+            for (String pair : queries) {
+                if (pair.contains("=")) {
+                    String[] qp = pair.split("=");
+
+                    if (qp[0].equals(LRAConstants.HTTP_METHOD_NAME)) {
+                        switch (qp[1]) {
+                            case "javax.ws.rs.GET":
+                                return asyncInvoker.get();
+                            case "javax.ws.rs.PUT":
+                                return asyncInvoker.put(Entity.text(null));
+                              //  return asyncInvoker.put(Entity.entity(compensatorData, mediaType));
```

Comment:
why we cannot already use compensatorData here? It's being used in default case (switch (method) {...}) later in this method

Reply:
Because it isn't supported in MP-LR milestone 1. We will re-enable it in the next one

Reply:
I though you are on PTO :) then go ahead and merge this please, it's passing TCK!! Great job!